data/influxdb_urls.yml

```diff
@@ -66,3 +70,9 @@ cloud:
         - name: East US (Virginia)
           location: Virginia, USA
           url: https://eastus-1.azure.cloud2.influxdata.com
+dedicated:
```

Comment:
Just a nit–in other places, we've used the key `cloud-dedicated` instead of just `dedicated`. InfluxDB Cloud Serverless just inherits data from `cloud`